packit/config/aliases.py

```diff
@@ -183,3 +210,38 @@ def get_koji_targets(*name: str, default=DEFAULT_VERSION) -> Set[str]:
 
 def get_all_koji_targets() -> List[str]:
     return run_command(["koji", "list-targets", "--quiet"], output=True).split()
+
+
+@functools.lru_cache(maxsize=None)
```

Comment:
[`maxsize=None` means](https://docs.python.org/3/library/functools.html#functools.lru_cache) the cache can grow without bound. Is that what we need here? I'd expect either `maxsize=1` or no `maxsize` (as I understand it the size would be 1 anyway since the ` get_aliases()` has no arguments).